src/libraries/System.Net.Quic/src/System.Net.Quic.csproj

```diff
@@ -129,7 +130,7 @@
 
   <!-- Support for deploying msquic -->
   <ItemGroup Condition="'$(TargetsWindows)' == 'true' and
-                        ('$(TargetArchitecture)' == 'x64' or '$(TargetArchitecture)' == 'x86')">
+                        ('$(TargetArchitecture)' == 'x64' or '$(TargetArchitecture)' == 'x86') and '$(DotNetBuildFromSource)' != 'true'">
     <BinPlaceDir Include="$(MicrosoftNetCoreAppRuntimePackNativeDir)" ItemName="NativeBinPlaceItem" />
```

Comment:
Not for this PR, but something to consider moving forward is that this fails silently if that quic package changes it's shape.  It might be good to avoid a wildcard here.  We should be *explicit* about things which we end up shipping in the shared framework.